modules/eso-plugins-config.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * security/external_secrets_operator/external-secrets-operator-api.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="eso-plugiins-config_{context}"]
+= pluginsConfig
+
+The `pluginsConfig` configures the optional plugins.
+
+[cols="1,1,1,1,1",options="header"]
+|===
+| Field
+| Type
+| Description
+| Default
+| Validation
+
+| `bitwardenSecretManagerProvider`
+| _object_
+| `bitwardenSecretManagerProvider` enables the `bitwarden-secrets-manager` provider plugin for connecting with the 'bitwarden-secrets-manager'.
```

Comment:
| `bitwardenSecretManagerProvider` enables the `bitwarden-secrets-manager` provider plugin for connecting with the `bitwarden-secrets-manager`.